lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -1885,9 +1885,9 @@ def model_get_baseline_system_type_by_zone(model, climate_zone, custom = nil)
           # separate the primary zones from the secondary zones.
           # Add the baseline system type to the primary zones
           # and add the suplemental system type to the secondary zones.
-          story_zone_lists.each do |zones|
+          story_zone_lists.each do |story_group|
             # Differentiate primary and secondary zones
-            pri_sec_zone_lists = model_differentiate_primary_secondary_thermal_zones(model, story_group, zone_fan_scheds)
+            pri_sec_zone_lists = model_differentiate_primary_secondary_thermal_zones(model, story_group)
```

Comment:
Unrelated undefined local variable found in openstudio-common-measures-gem `GenericQAQC_Test#test_GenericQAQC_pass` throws

```
undefined local variable or method `story_group' for #<ASHRAE9012013:0x0000561f90a41108>)
/home/julien/Software/Others/openstudio_gems/openstudio-standards/lib/openstudio-standards/standards/Standards.Model.rb:1890:in `block (2 levels) in model_get_baseline_system_type_by_zone'
```

Also

```
undefined local variable or method `zone_fan_scheds' for #<ASHRAE9012013:0x0000564cbad042a8>
```

It's been there for more than 2 years though? https://github.com/NREL/openstudio-standards/blob/91b54056f8dd345589079f1c24abad51d14e6d10/lib/openstudio-standards/standards/Standards.Model.rb#L2191-L2193